demos/common/python/pipelines/async_pipeline.py

```diff
@@ -27,6 +88,10 @@ def __init__(self, ie, model, plugin_config, device='CPU', max_num_requests=1):
         self.logger.info('Loading network to {} plugin...'.format(device))
         self.exec_net = ie.load_network(network=self.model.net, device_name=device,
                                         config=plugin_config, num_requests=max_num_requests)
+        if max_num_requests == 0:
+            # +1 to use it as a buffer of the pipeline
+            self.exec_net = ie.load_network(network=self.model.net, device_name=device,
+                                            config=plugin_config, num_requests=len(self.exec_net.requests) + 1)
```

Comment:
Why do you load net twice? May  be `if ... else ...` more suitable here.

Reply:
There is no other way to query a suggestion about nireq in python

Reply:
You mean with `num_requests=0` the IE will determine it by d20? Ok, please add comment to this odd entity.

Reply:
What is d20? Added

Reply:
is it lack of functionality in python api? is it make sense to submit bug on this?

Reply:
It is well described: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/ie_bridges/python/src/openvino/inference_engine/ie_api.pyx#L972 so it's not a bug, but lack of functionality.

Reply:
@ilya-lavrenov is it ok, to require customer to call load_network twice to query suggested nireq ?

Reply:
CC @akuporos @nkogteva 

I don't think that it's OK (at least C++ allows to create as many infer requests as you can), but looks like it's python limitations.